packages/generator/src/operations/request-builder-arguments.ts

```diff
@@ -15,15 +15,17 @@ export function getRequestBuilderArgumentsBase(
       `'${service.serviceOptions.basePath}'`,
       `'${operation.originalName}'`,
       ...sharedParameters,
-      'deSerializers'
+      'deSerializers',
+      `'${operation.type}'`
```

Comment:
[req] Seems that this is the reason we are removing this parameter in the only place where `getRequestBuilderArgumentsBase()` is called. I guess we could just not add it here 😊

Reply:
This extra parameter is added here because we need to add this `type` to the generated OperationRequestBuilder function in test-packages, i.e.
```
const returnStatement = `return new ${requestBuilderName}(${parameters.join(
    ', '
  )});`;
```

Reply:
I can't find this, can you point out where I can find this code?